docs/integrations/saas-cloud/dropbox.md

```diff
@@ -116,21 +116,15 @@ _sourceCategory="dropboxSource"
 
 This section provides instructions for setting up [Cloud-to-Cloud-Integration for Dropbox Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dropbox-source) to create the source and use the same source category while installing the app.
 
-## Installing the Dropbox app​
+## Installing the Dropbox app
 
-This section has instructions for installing the Dropbox App for Sumo Logic.
+{@import ../../reuse/apps/app-install-v2.md}
 
-{@import ../../reuse/apps/app-install.md}
+## Viewing Dropbox dashboards
 
-## Viewing the Dropbox Dashboards​
+{@import ../../reuse/apps/view-dashboards.md}
 
-* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
-
-You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
-
-* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
-
-###  Dropbox Dashboard Overview
+###  Overview
 
 **Dropbox - Overview**.
```

Comment:
It looks like the Dashboard names are repeated (2x) in this doc. For example....

```
### Overview
**Dropbox - Overview**.

### File Statistics
**Dropbox - File Statistics**.
```

Reply:
Updated the Doc